cli-commands/forgae-deploy/forgae-deployer.js

```diff
@@ -99,10 +109,70 @@ class Deployer {
         }
 
         logStoreService.logAction(info);
-
         console.log(`===== Contract: ${contractFileName} has been deployed =====`)
         return deployedContract;
+
+        function addFromFunction (obj) {
+
+            const additionalFunctionality = {
+                from: function (secretKey) {
+
+                    if (!secretKey || !isNaN(secretKey) || secretKey.length !== 128) {
+                        throw new Error('Invalid secret key!');
+                    }
+
+                    return {
+                        call: async function (functionName, options) {
+
+                            const keyPair = await generateKeyPairFromSecretKey(secretKey);
+                            const newClient = await utils.getClient(self.network, keyPair);
+
+                            const anotherClientConfiguration = {
+                                client: newClient,
+                                byteCode: compiledContract.bytecode,
+                                contractAddress: obj.address
+                            }
+
+                            let configuration = {
+                                options: {
+                                    ttl: ttl
+                                },
+                                abi: ABI_TYPE,
+                            };
+                        
+                            if (options.args) {
+                                configuration.args = options.args
+                            }
+
+                            if (options.amount && options.amount > 0) {
+                                configuration.options.amount = options.amount;
+                            }
+                        
+                            return await anotherClientConfiguration.client.contractCall(anotherClientConfiguration.byteCode, ABI_TYPE, anotherClientConfiguration.contractAddress, functionName, configuration);
+                        }
+                    }
+                }
+            }
+
+            const newObj = Object.assign(additionalFunctionality, obj);
```

Comment:
use a more accurate name than "newObj". What is this object representing?

Reply:
if I accept ANY object and create copy of them is this new object?

Reply:
Like at the top, this is probably some kind of an instance of something, probably something related to contracts will be better for a name. What do you think